docs/src/core/reference/python/io.rst

```diff
@@ -3,24 +3,32 @@
 Serialization
 ^^^^^^^^^^^^^
 
-.. autofunction:: metatensor.load
-
 .. autofunction:: metatensor.save
 
-.. autofunction:: metatensor.load_labels
+.. autofunction:: metatensor.load
 
---------------------------------------------------------------------------------
+.. autofunction:: metatensor.load_labels
 
-.. autofunction:: metatensor.io.load_buffer
+.. autofunction:: metatensor.load_block
```

Comment:
Super picky. In the docs we follow the order `TensorMap`, `TensorBlock`, `Labels`

![Screenshot 2024-08-28 at 11 11 52](https://github.com/user-attachments/assets/87a3930e-9e5c-4f19-8315-5547e6646f48)

Should we also follow this order here?

Reply:
sure!